storybook/storybook-react/src/Grid/Grid.stories.tsx

```diff
@@ -31,66 +68,96 @@ const StoryTemplate: Story = {
   ],
 }
 
-export const Default: Story = {
-  ...StoryTemplate,
+const GridCellStoryTemplate: GridCellStory = {
+  argTypes: gridCellArgTypes,
+  decorators: [
+    (Story) => (
+      <Screen>
+        <Grid>
+          <Story />
+        </Grid>
+      </Screen>
+    ),
+  ],
+  render: ({ children, ...args }) => <Grid.Cell {...args}>{children}</Grid.Cell>,
+}
+
+export const Default: GridStory = {
+  ...GridStoryTemplate,
   args: {
     children: Array.from(Array(12).keys()).map((i) => <Grid.Cell className="amsterdam-docs-pink-box" key={i} />),
```

Comment:
We're using this same array thrice, can't we add it to the `GridStoryTemplate`?

Reply:
Seems logical, but results in the second and third canvas being empty and showing incorrect code. I tried extracting a variable and a function as well. Can you give it a shot?

Reply:
Updated.